### DISCUSSION TITLE
zh/guide/layout/page

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#481] [Announcements]
# zh/guide/layout/page

图标支持 你可以在页面的 frontmatter 中配置 icon 字段，填入对应图标的 FontClass 即可绑定图标到页面。 该图标会在导航栏，侧边栏，路径导航和标题中使用。 例子 图标的相关设置，详见 图标支持 (../interface/icon.md) 路径导航 详见 路径导航 (breadcrumb.md) 章节。 文章信息展示 详见 文...

https://theme-hope.vuejs.press/zh/guide/layout/page.html

**Comment by VivaAlexYu** [comment]
评估下右侧的标题列表增加标题深度配置项，可以显示多级目录标题，目前智能显示到3级目录，谢谢。
